frontend/src/components/dashlist/FlakinessTable.jsx

```diff
@@ -50,10 +49,15 @@ const DashboardList = () => {
     }, [dispatch, seriesId, stabilityChecker, amountOfBuilds, offset]);
 
     return (
-        <TableContainer id="flakiness-table">
-            <StabilityButton value={'unstable'} text="Unstable" />
-            <StabilityButton value={'stable'} text="Stable" />
-            <StyledTable>
+        <TableContainer>
+            <p className="sr-show" role="status" id="flakiness-table-status">
+                {' '}
+            </p>
+            <span role="radiogroup" aria-controls="flakiness-table">
+                <StabilityButton value={'unstable'} text="Unstable" />
```

Comment:
The focus style for unstable doesnät show in left side of the button 
![image](https://user-images.githubusercontent.com/17247485/111778293-1fe17d80-88bd-11eb-8bcb-f93c2946fece.png)

Reply:
Same goes here, I think this should be fixed in another issue where we style the whole table :)